src/app/content/search/components/SearchResultsSidebar/__snapshots__/index.spec.tsx.snap

```diff
@@ -158,7 +158,7 @@ exports[`SearchResultsSidebar matches snapshot with results 1`] = `
       <a
         className="styled__SectionContentPreview-as3fh1-11 dMNXGN"
         data-testid="search-result"
-        href="books/book-slug-1/pages/test-page-1"
+        href="books/book-slug-1/pages/test-page-1?archive=https://openstax.org/apps/archive/20201029.133542"
```

Comment:
these shouldn't have changed, contentlink is adding it because it thinks it has to, your mocks probably need work